cmd/cli/license/inspect.go

```diff
@@ -0,0 +1,207 @@
+}
+
+// Update table columns
+var licenseProductColumn = output.TableColumn[LicenseInfo]{
+	ColumnConfig: table.ColumnConfig{Name: "PRODUCT"},
+	Value:        func(l LicenseInfo) string { return l.Product },
+}
+
+var licenseLicenseIDColumn = output.TableColumn[LicenseInfo]{
+	ColumnConfig: table.ColumnConfig{Name: "LICENSE ID"},
+	Value:        func(l LicenseInfo) string { return l.LicenseID },
+}
+
+var licenseCustomerIDColumn = output.TableColumn[LicenseInfo]{
+	ColumnConfig: table.ColumnConfig{Name: "CUSTOMER ID"},
+	Value:        func(l LicenseInfo) string { return l.CustomerID },
+}
+
+var licenseValidUntilColumn = output.TableColumn[LicenseInfo]{
+	ColumnConfig: table.ColumnConfig{Name: "VALID UNTIL"},
+	Value:        func(l LicenseInfo) string { return l.ValidUntil },
+}
+
+var licenseLicenseVersionColumn = output.TableColumn[LicenseInfo]{
+	ColumnConfig: table.ColumnConfig{Name: "VERSION"},
+	Value:        func(l LicenseInfo) string { return l.LicenseVersion },
+}
+
+var licenseCapabilitiesColumn = output.TableColumn[LicenseInfo]{
+	ColumnConfig: table.ColumnConfig{Name: "CAPABILITIES"},
+	Value: func(l LicenseInfo) string {
+		if len(l.Capabilities) == 0 {
+			return "none"
+		}
+		var caps []string
+		for k, v := range l.Capabilities {
+			caps = append(caps, fmt.Sprintf("%s=%s", k, v))
+		}
+		return strings.Join(caps, ", ")
+	},
+}
+
+func (o *InspectOptions) Run(ctx context.Context, cmd *cobra.Command) error {
+	// Read the license file
+	data, err := os.ReadFile(o.LicenseFile)
+	if err != nil {
+		return fmt.Errorf("failed to read license file: %w", err)
+	}
+
+	// Parse the license file
+	var license licenseFile
+	if err := json.Unmarshal(data, &license); err != nil {
+		return fmt.Errorf("failed to parse license file: %w", err)
+	}
+
+	// Create offline license validator
+	validator, err := licensepkg.NewOfflineLicenseValidator()
+	if err != nil {
+		return fmt.Errorf("failed to create license validator: %w", err)
+	}
+
+	// Validate the license token
+	claims, err := validator.ValidateToken(license.License)
+	if err != nil {
+		return fmt.Errorf("invalid license: %w", err)
+	}
+
+	// Create license info from the validated claims
+	licenseInfo := LicenseInfo{
+		Product:        claims.Product,
+		LicenseID:      claims.LicenseID,
+		CustomerID:     claims.CustomerID,
+		ValidUntil:     claims.ExpiresAt.Format(time.DateOnly),
+		LicenseVersion: claims.LicenseVersion,
+		Capabilities:   claims.Capabilities,
+	}
+
+	// For JSON/YAML output, wrap the license info with metadata
+	if o.OutputOpts.Format == output.JSONFormat || o.OutputOpts.Format == output.YAMLFormat {
+		return output.OutputOne(cmd, nil, o.OutputOpts, jsonOutput{
+			LicenseInfo: licenseInfo,
+			Metadata:    claims.Metadata,
+		})
+	}
+
+	columns := []output.TableColumn[LicenseInfo]{
+		licenseProductColumn,
+		licenseLicenseIDColumn,
+		licenseCustomerIDColumn,
+		licenseValidUntilColumn,
+		licenseLicenseVersionColumn,
+		licenseCapabilitiesColumn,
+	}
```

Comment:
I don't see the value for a tabular format, at least not when inspecting a single license. The default should be either a key/value formatted output like `bacalhau job describe` or just yaml

```
→ bacalhau job describe j-b9d12ae6
ID            = j-b9d12ae6-c40c-4eaf-bcf5-0f3e1106b2f4
Name          = j-b9d12ae6-c40c-4eaf-bcf5-0f3e1106b2f4
Namespace     = default
Type          = batch
State         = Completed
Count         = 1
Created Time  = 2025-01-13 13:51:50
Modified Time = 2025-01-13 13:51:51
Version       = 0
...
```

Reply:
@wdbaruni I am in favor of consistency among other other commands.
If key/value formatted output is the more prominent output, I'll go for it. I chose the tabular format since I noticed it is more common with bacalhau commands outputs, so I went with it for a better user experience.

If the you prefer `key/value formatted` output, I'll go for it.
Let me know.

Reply:
@wdbaruni changed to us key/value formatted